scripts/performance.sh

```diff
@@ -14,6 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if [[ "$1" == "--dev" || "$1" == "-d" ]]; then
+ export PERFORMANCE_DEV=1
+ pubs = $2
+ echo $2
+ export PUBLISHES=${2:-1000}
```

Comment:
Did you mean:
```suggestion
 export PUBLISHES=${pubs:-1000}
```

Reply:
I meant `2`, just to take the second argument